modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/nhttp/ClientHandler.java

```diff
@@ -158,6 +158,18 @@
     private static final String CONTENT_TYPE = "Content-Type";
 
     /**
+     * Configuration which decides if message size validation is enabled/disabled. If enabled, messages of a size
+     * greater than 'validMaxMessageSize' will be dropped.
+     */
+    private static boolean isMessageSizeValidationEnabled = false;
```

Comment:
Do we really need two properties here? IMO, validMaxMessageSize is sufficient. If it is not there, simply this feature is disabled. WDYT?